dependencies/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+
+include(FetchContent)
```

Comment:
You need at least CMake 3.11 to use FetchContent. The reason it doesn't fail locally is because you have a newer version, but travis doesn't have. 
In order to align with sqlpp, please bump up the version in CMakeLists.txt to 3.14. Additionally you will also need to update the travis file, so it uses a more recent cmake version. It should be sufficient to bump up "dist" to focial in .travis.yml